docs/user-manual/address-settings.adoc

```diff
@@ -394,6 +394,10 @@ that helps to detect and prevent the processing of duplicate messages based on t
 By default, the `id-cache-size` setting inherits from the global `id-cache-size`, with a default of `20000`
 elements if not explicitly configured. Read more about xref:duplicate-detection.adoc#configuring-the-duplicate-id-cache[duplicate id cache sizes].
 
+intermediate-message-buffer-initial-size::
+defines the initial number of elements of the intermediate message reference buffer (an `Object[]`), which is instantiated for each queue.
+By default, this value is `8192` bytes if not explicitly configured. This must be a positive power of 2 (i.e. `0` is not an option).
```

Comment:
The units should be removed since it isnt in bytes: "value is `8192` _bytes_"

Dont think the array type needs listed, doesn't seem needed for the usage and just seems likely to go stale at some point.

Reply:
Done